src/crimson/os/seastore/transaction.h

```diff
@@ -541,6 +554,8 @@ class Transaction {
   std::list<CachedExtentRef> inline_block_list;
   /// blocks that will be committed with out-of-line record
   std::list<CachedExtentRef> written_ool_block_list;
+  /// delta overwrite blocks that will be committed with mutated out-of-line record
+  std::list<CachedExtentRef> inplace_delta_overwrite_ool_block_list;
```

Comment:
These lists start to confuse me, still trying to understand if there is a way to simplify.

Reply:
Since we don't need to allocate additional block (the blocks already exists in RBM), the only thing we need to do here is adding delta for inplace-delta-overwrite to the transaction. Consequently, `transaction_manager` handles this at `do_submit_transaction`.

Reply:
Is `written_inplace_ool_block_list` a better name here according to `written_ool_block_list`?

Reply:
Done.